### PR TITLE
Awesome books: plain JavaScript with objects

### PR DESCRIPTION
**Pull request by safafa**
In this pull request we :

- Created a list of Books using javascript.
- The user can add a new book.
- The user can remove a book from the list and the list will update.
- We made use of localStorage where we stored the data about the books.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+const store = window.localStorage;
+const list = document.getElementById('booklist');
+const form = document.getElementById('form');
+
+function displayBooks() {
+  while (list.firstChild) {
+    list.removeChild(list.firstChild);
+  }
+  books = JSON.parse(localStorage.getItem('storeBook'));
+  books.forEach((book) => {
```
Comment (carlos-ssh) [inline script.js:11]: **Kindly fix these errors that appear in console:**
Screen Shot 2021-08-10 at 04 59 38

**When a user clicks the "Add" button:**
1. A new book is added to the collection. 
2. The new book is displayed on the page.


**When a user clicks the "Remove" button:**
1. a. The right book is removed from the collection. 
2. b. The right book disappears from the page.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hello Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Add the right [Linters](https://github.com/microverseinc/linters-config/tree/master/html-css-js) into your project.

File: script.js (hunk 1 above)
Comment (safafa) [reply script.js:11]: > 
> 
> **Kindly fix these errors that appear in console:**
> Screen Shot 2021-08-10 at 04 59 38
> 
> **When a user clicks the "Add" button:**
> 
>     1. A new book is added to the collection.
> 
>     2. The new book is displayed on the page.
> 
> 
> **When a user clicks the "Remove" button:**
> 
>     1. a. The right book is removed from the collection.
> 
>     2. b. The right book disappears from the page.

Hello Thanks for the review. I checked on my side it's working. I have no error in the console.
image

**Review by bellom — APPROVED** [review]
## Hi Team 👏  

**Great work making all the necessary corrections from the previous review.**

**It would be nice to create a function for both `Add` and `Remove` as stated in the requirement, doing it that way, your code will be more organized and clean.**
Project Requirement Page:
image

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**